connection.js

```diff
@@ -74,6 +75,7 @@ function checkForAttachParametersAndConnect(id, password, connection) {
 function connect(id, password, roomName) {
     const connectionConfig = Object.assign({}, config);
     const { issuer, jwt } = APP.store.getState()['features/base/jwt'];
+    const { locationURL } = APP.store.getState()['features/base/connection'];
```

Comment:
maybe extract the state to a variable?

Reply:
will do, I was only worried this could encourage to re-use the var from the callbacks below and lead to bugs